CONTRIBUTORS

```diff
@@ -29,6 +29,7 @@ Christophe Meessen <christophe.meessen@gmail.com>
 Christopher Waldon <christopher.waldon.dev@gmail.com>
 Clayton Northey <clayton.northey@gmail.com>
 Dan Kortschak <dan.kortschak@adelaide.edu.au> <dan@kortschak.io>
+Dan Kortschak <lorenc.d@gmail.com>
```

Comment:
```suggestion
Dan Lorenc <lorenc.d@gmail.com>
```
?

Reply:
D'oh! You beat me before I fixed it myself :)